modules/xmpp/ChatRoom.js

```diff
@@ -190,12 +190,12 @@ export default class ChatRoom extends Listenable {
 
             // there is no point of sending conference iq when in visitor mode
             const preJoin
-                = this.options.disableFocus || this.options.hosts.visitorFocus
+                = this.options.disableFocus || this.xmpp.options.hosts.visitorFocus
```

Comment:
I don't think that is correct. 
Can you explain what is the problem you face? 
There is issue with that code, but the fix was just merged https://github.com/jitsi/lib-jitsi-meet/commit/8acdeb1ddc30e665b3de995ace893630abda45c7

Reply:
I'm guessing you were hitting the same ...

Reply:
Ah that's the exact issue I was experiencing! Thank you